### PR TITLE
[AF-1489] Set up fossa open source scanning

### PR DESCRIPTION
**Pull request by token-cjg**
Set up Fossa open source scanning.

Don't forget to floss!

## References

* https://zendesk.atlassian.net/browse/AF-1489

### CODE REVIEW

**Comment by token-cjg** [comment]
@zendesk/vegemite

File: .travis.yml [hunk 1]
```diff
@@ -11,9 +11,17 @@ cache:
     - node_modules
 branches:
   only: [master]
-jobs:
+matrix:
   include:
     - stage: Tests
```
Comment (hoangtrvu) [inline .travis.yml]: Do we need this line `stage: Tests`? I was comparing this with `.travis.yml` in ZAF [here](https://github.com/zendesk/zendesk_app_framework/blob/master/.travis.yml)

File: README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ Zendesk App Framework SDK
 =========================
 
 [![Build Status](https://travis-ci.org/zendesk/zendesk_app_framework_sdk.svg?branch=master)](https://travis-ci.org/zendesk/zendesk_app_framework_sdk)
+[![FOSSA Status](https://app.fossa.io/api/projects/custom%2B4071%2Fgithub.com%2Fzendesk%2Fzendesk_app_framework_sdk.svg?type=shield)](https://app.fossa.io/projects/custom%2B4071%2Fgithub.com%2Fzendesk%2Fzendesk_app_framework_sdk?ref=badge_shield)
```
Comment (hoangtrvu) [inline README.md]: The embed code that I get is slightly different from this ``[FOSSA Status](https://app.fossa.io/projects/git%2Bgithub.com%2Fzendesk%2Fzendesk_app_framework_sdk?ref=badge_shield)``, not sure why though.

Comment (token-cjg) [reply README.md]: Yes, I didn't check this 😥

File: .travis.yml (hunk 1 above)
Comment (token-cjg) [reply .travis.yml]: No, I don't think we do!

**Pull request merged**